### ISSUE TITLE
provide an api for i18n

### ISSUE THREAD

**Issue by MiniDigger** [Proximyst/SpigotVult#45]

**Comment by Proximyst** [comment]
I'm assuming you want something like Essentials' I18n module? https://github.com/essentials/Essentials/blob/2.x/Essentials/src/com/earth2me/essentials/I18n.java

**Comment by MiniDigger** [comment]
yes. there are thousands of ways to implement that. I think essentials approach (using java resource bundles) is sane. 
an idea on how to implement would be requirering plugins that want to use this feature to ship a en_EN.properties in the root of the jar that then could automatically be saved to the plugin folder and loaded from there.

**Comment by Proximyst** [comment]
@MiniDigger I'd rather keep the required resources to a minimum, myself. 
A normal circumstance for not wanting this is just having a plugin for your own server, including small messages. 
This'll just be a hassle to update, when all the plugin does is for example some staff menu.
It could just have one message saying "Das Menü war geöffnet." or something like that. (note I am not godo in German at all)
